### DISCUSSION TITLE
CP Kurs-Gen-Einstellung mit manchen Tracktoren nicht vorhanden

### DISCUSSION THREAD

**Discussion by AdvocatusDei** [Courseplay/Courseplay_FS22#3040] [Q&A] [answered]
Moin, 

ich nutzte jetzt seit ca 5 Tagen erst CP, heute aber ist mir aufgefallen das CP nicht so will wie ich möchte. 

Ich wollte Kalk ausbringen mit meinem MF 6S und 7S und dem Bredal Kalkstreuer.
Bin zum Feld gefahren machte den CP Helfer auf. Hab das Feld markiert habe dann die SP Kurs-Gen-Einstellungen geöffnet und sie da, geht nicht so wie ich will. Kein Vorgewende, keine Überlappung, keine Arbeitsbreite etc etc. Siehe Screenshot.
Hab mir dann nen Fendt 724 gekauft und hingefahren das selbe. Häng ich jetzt meinen MF 8S davor gehts. Da wird mir dann in den CP-Kurs-Gen-Einstellungen alles angezeigt. Woran könnte das liegen. Log ist angefügt und auch ein Screenshot vom Fehler.

![fsScreen_2023_12_30_00_35_52](https://github.com/Courseplay/Courseplay_FS22/assets/29044006/920abb02-0f3f-446f-a406-4d66e2c5dc4b)
[log.txt](https://github.com/Courseplay/Courseplay_FS22/files/13796514/log.txt)

**Comment by BalticHawk** [comment, marked as answer]
Du musst einfach mit dem Mausrad nach oben scrollen.

**Reply by AdvocatusDei** [reply]
In dem Fenster gibts nix zu scrollen! Heute morgen geht es aber wie aus Geisterhand. WTF

**Reply by Tensuko** [reply]
Doch, einfach mit dem Mauszeiger rein gehen und scrollen, sonst würdest du auch bei anderen Geräten nicht alle Einstellungen sehen können.

**Reply by AdvocatusDei** [reply]
In dem Fenster gibt es nichts zum Scrollen das sind die einzigen Einträge die da sind. Sonst hätte ich wohl kaum diesen Bug hier beschrieben oder?
